src/components/base/JInput/JInput.js

```diff
@@ -49,10 +53,28 @@ class JInput extends PureComponent<Props, *> {
       infoMessage,
       errorMessage,
       isLoading,
-      isChecked,
       isDisabled,
+      isPinCode,
+      isMultiline,
+      render,
     } = this.props
+
     const labelOrPlaceholder = label || placeholder
+    const hasTopLabel = (color === 'gray') && labelOrPlaceholder
+    const placeholderFromI18n =
+            (labelOrPlaceholder && i18n(labelOrPlaceholder)) || labelOrPlaceholder
```

Comment:
too many spaces from start of line

Reply:
weird it looks fine in webstorm
<img width="654" alt="screen shot 2018-10-09 at 08 15 07" src="https://user-images.githubusercontent.com/490365/46647959-74845880-cb9b-11e8-8a88-f35e866ae8fb.png">

but would fix, let's move forward